app/views/collections/_form.html.erb

```diff
@@ -1,11 +1,19 @@
 <%= simple_form_for [collections, @form], html: {class: 'form-horizontal editor'} do |f| %>
   <div id="descriptions_display">
```

Comment:
I will double check with @leahvanderjagt but I think we don't need a button, but just to have the description displayed on the page.